lib/dotcom/trip_plan/itinerary_row_list.ex

```diff
@@ -4,6 +4,7 @@ defmodule Dotcom.TripPlan.ItineraryRowList do
 
   An optional to and from name can be passed in.
   """
+  alias Dotcom.TripPlan.Alerts, as: TripPlanAlerts
```

Comment:
This is just one example, but the same applies to all. `Alerts` should be added to the excluded lastnames list because `TripPlan.Alerts` is more clear than `TripPlanAlerts.`